.NET/Microsoft.Recognizers.Definitions.Common/BaseURL.cs

```diff
@@ -23,7 +23,7 @@ public static class BaseURL
 	{
 		public const string ProtocolRegex = @"((https?|ftp):\/\/)";
 		public const string PortRegex = @"(:\d{1,5})";
-		public const string ExtractionRestrictionRegex = @"(?<=\s|[\'""\(\[:]|^)";
+		public const string ExtractionRestrictionRegex = @"(?<=\s|[\'""\(\[:]|[\u4E00-\u9FFF]|^)";

```

Comment:
What's the goal of this change? By design we don't support unicode in URLs (unless there is a very strong reason to add it).
There is also no new spec in this PR that relates to it. Any new fix or extension in code needs to at least have a new spec to show that works.